### PR TITLE
Adds a collection to show ws usage with custom resource

### PR DESCRIPTION
**Pull request by thomasnares**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Adds a collection to show ws usage with custom resource
| Type?             | new feature

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
-# webservice-postman-examples
+# Webservice Postman examples
+
+This repository is showing some Postman collections to illustrate how to use PrestaShop's webservices.
```
Comment (kpodemski) [inline README.md]:
```suggestion
This repository shows some Postman collections to illustrate how to use PrestaShop's webservices.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+
+## What is Postman
+
+Postman is a powerful collaboration platform that simplifies the process of developing, testing, documenting, and sharing APIs (Application Programming Interfaces) by providing a user-friendly interface and a comprehensive set of tools.
+
+As a developer, Postman can greatly assist you in various ways. It allows you to effortlessly send HTTP requests to APIs and receive responses, enabling you to test and debug your own APIs or interact with external APIs. Postman supports different request methods, request headers, query parameters, and request bodies, making it easy to explore API endpoints and their functionalities.
```
Comment (kpodemski) [inline README.md]:
```suggestion
As a developer, Postman can greatly assist you in various ways. It allows you to effortlessly send HTTP requests to APIs and receive responses, enabling you to test and debug your own APIs or interact with external APIs. Postman supports different request methods, request headers, query parameters, and request bodies, making it explore API endpoints and their functionalities.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,17 @@
...
+
+More informations: [Official Postman website](https://www.postman.com/)
+
+## Examples
+
+### demo_custom_resource_collection.json
+
+This collection illustrates all the common CRUD actions on a custom resource (articles), from the example module [demowsextend](https://github.com/PrestaShop/example-modules/tree/master/demowsextend), and explained here : [extend webservice with custom resource in devdocs](https://devdocs.prestashop-project.org/8/modules/concepts/webservice/).
```
Comment (kpodemski) [inline README.md]:
```suggestion
This collection illustrates all the common CRUD actions on a custom resource (articles) from the example module [demowsextend](https://github.com/PrestaShop/example-modules/tree/master/demowsextend), and explained here  [extend webservice with the custom resource in devdocs](https://devdocs.prestashop-project.org/8/modules/concepts/webservice/).
```

**Review by kpodemski — COMMENTED** [review]
A few comments but overall LGTM 🎉

File: README.md (hunk 3 above)
Comment (thomasnares) [inline README.md]:
```suggestion
This collection illustrates all the common CRUD actions on a custom resource (articles) from the example module [demowsextend](https://github.com/PrestaShop/example-modules/tree/master/demowsextend), and explained here [extend webservice with custom resource in devdocs](https://devdocs.prestashop-project.org/8/modules/concepts/webservice/).
```

File: README.md [hunk 4]
```diff
@@ -1 +1,17 @@
...
+
+As a developer, Postman can greatly assist you in various ways. It allows you to effortlessly send HTTP requests to APIs and receive responses, enabling you to test and debug your own APIs or interact with external APIs. Postman supports different request methods, request headers, query parameters, and request bodies, making it explore API endpoints and their functionalities.
+
+More informations: [Official Postman website](https://www.postman.com/)
+
+## Examples
+
+### demo_custom_resource_collection.json
+
+This collection illustrates all the common CRUD actions on a custom resource (articles) from the example module [demowsextend](https://github.com/PrestaShop/example-modules/tree/master/demowsextend), and explained here [extend webservice with custom resource in devdocs](https://devdocs.prestashop-project.org/8/modules/concepts/webservice/).
```
Comment (thomasnares) [inline README.md]:
```suggestion
This collection illustrates all the common CRUD actions on a custom resource (articles) from the example module [demowsextend](https://github.com/PrestaShop/example-modules/tree/master/demowsextend), and explained here [extend webservice with the custom resource in devdocs](https://devdocs.prestashop-project.org/8/modules/concepts/webservice/).
```

**Comment by kpodemski** [comment]
@thomasnares could you add a little tutorial how to use this json?

**Comment by thomasnares** [comment]
yes sure 👍

**Comment by thomasnares** [comment]
@kpodemski lets merge it so i can link to it in devdocs, and i will add a short note here on how to use the json

**Review by mflasquin — APPROVED** [review]

**Pull request merged**